### PR TITLE
Document CoC Member selection

### LINKED ISSUE #1297: Document CoC Team selection process
The [CoC Team Charter](https://github.com/openjs-foundation/cross-project-council/blob/main/conduct/COC_TEAM_CHARTER.md#member-selection) is voluntarily non-prescriptive when it comes to selecting CoC Team members.

Our plan, discussed in the working session of April 23, is to document a process here as we run the first selection and formalize for next year.

Ideally it's lightweight, and close to other election mechanisms.

Consensus in the work session was to make it consensus-driven rather than an election process (this echoes -1 being a a blocker expressed in prior issues opened on the topic).

### PR DESCRIPTION
**Pull request by joesepi**
closes #1297

### CODE REVIEW

**Review by mhdawson — APPROVED** [review]
LGTM

File: conduct/COC_TEAM_CHARTER.md [hunk 1]
```diff
@@ -45,7 +45,9 @@ The Cross Project Council (CPC) will hold an annual selection process in order t
 
 The CPC is responsible for ensuring that the CoC Team is as representative and diverse as possible. To do so, the CPC commits to reach out proactively to individuals from underrepresented groups.
 
-Candidates can self-nominate to be selected as an alternate community member, a primary member, or both.
+Candidates can self-nominate to be selected as an alternate community member, a primary member, or both. If there is an objection to a nomination and consensus cannot be reached on that nomination then a vote will occur on that nomination.
```
Comment (tobie) [inline conduct/COC_TEAM_CHARTER.md:48]:
```suggestion
Candidates can self-nominate to be selected as an alternate community member, a primary member, or both.
```

File: conduct/COC_TEAM_CHARTER.md [hunk 2]
```diff
@@ -45,7 +45,9 @@ The Cross Project Council (CPC) will hold an annual selection process in order t
 
 The CPC is responsible for ensuring that the CoC Team is as representative and diverse as possible. To do so, the CPC commits to reach out proactively to individuals from underrepresented groups.
 
-Candidates can self-nominate to be selected as an alternate community member, a primary member, or both.
+Candidates can self-nominate to be selected as an alternate community member, a primary member, or both. If there is an objection to a nomination and consensus cannot be reached on that nomination then a vote will occur on that nomination. 
+
+Once the nomination period has ended, primary and alternate members will be selected based on consensus (and voting if necessary). Our policy on consensus and voting is outlined in our [Charter](https://github.com/openjs-foundation/cross-project-council/blob/main/CPC-CHARTER.md#section-9-decision-making).
```
Comment (tobie) [inline conduct/COC_TEAM_CHARTER.md:50]:
```suggestion
Once the nomination period has ended, primary and alternate members will be selected based on consensus. If there is an objection to a nomination and consensus cannot be reached then a vote will occur for that nomination. Our policy on consensus and voting is outlined in our [Charter](https://github.com/openjs-foundation/cross-project-council/blob/main/CPC-CHARTER.md#section-9-decision-making). 
```

**Review by tobie — APPROVED** [review]
This LGTM with or without my suggestion (which essentially reorders sentences and avoids duplication).

The diff of diff makes my suggestions look weird. He's what I had in mind:

> Candidates can self-nominate to be selected as an alternate community member, a primary member, or both.
> 
> Once the nomination period has ended, primary and alternate members will be selected based on consensus. If there is an objection to a nomination and consensus cannot be reached then a vote will occur for that nomination. Our policy on consensus and voting is outlined in our [Charter](https://github.com/openjs-foundation/cross-project-council/blob/main/CPC-CHARTER.md#section-9-decision-making).

**Comment by tobie** [comment]
After further offline discussion with @joesepi, I'd go with something along the lines of:

> Candidates can self-nominate to be selected as an alternate community member, a primary member, or both.
> 
> Once the nomination period has ended, primary and alternate members will be selected based on consensus during a private CPC meeting. If there is an objection to a candidate being selected and consensus cannot be reached then a vote will occur for that candidate. Our policy on consensus and voting is outlined in our [Charter](https://github.com/openjs-foundation/cross-project-council/blob/main/CPC-CHARTER.md#section-9-decision-making).

File: conduct/COC_TEAM_CHARTER.md [hunk 3]
```diff
@@ -47,6 +47,10 @@ The CPC is responsible for ensuring that the CoC Team is as representative and d
 
 Candidates can self-nominate to be selected as an alternate community member, a primary member, or both.
 
+Once the nomination period has ended, primary and alternate members will be selected through [consensus][decision-making] during a private CPC meeting. If there is an objection to a candidate being selected and consensus cannot be reached, a [vote][voting] will occur for that candidate.
+
+[voting]: https://github.com/openjs-foundation/cross-project-council/blob/main/CPC-CHARTER.md#section-10-voting
```
Comment (joesepi) [inline conduct/COC_TEAM_CHARTER.md:52]:
```suggestion
[decision-making]: https://github.com/openjs-foundation/cross-project-council/blob/main/CPC-CHARTER.md#section-9-decision-making
[voting]: https://github.com/openjs-foundation/cross-project-council/blob/main/CPC-CHARTER.md#section-10-voting
```

**Comment by joesepi** [comment]
I worked through adding your suggestions, @tobie, please re-review.

@mhdawson please re-review as well. Thanks!

**Comment by tobie** [comment]
Shouldn't we also repurpose the forms and process that @bensternthal uses for the other elections here? (I think we discussed preserving candidate's privacy during the first part of the selection process, but I'm not sure where we landed with this.)

**Review by tobie — APPROVED** [review]

**Review by mhdawson — APPROVED** [review]

**Review by mertcanaltin — APPROVED** [review]

**Review by mcollina — APPROVED** [review]
lgtm

**Review by UlisesGascon — APPROVED** [review]

**Pull request merged**